### PR TITLE
#3166

### PR DESCRIPTION
**Pull request by vestrada**
https://github.com/craftercms/craftercms/issues/3166 - [studio-ui] let user know they don't have access to any sites when sites dashboard is empty #3166

### CODE REVIEW

File: static-assets/scripts/resources/locale-es.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
       "ACCOUNT": "Cuenta",
       "SITE_ERROR" : "es un sitio inválido",
       "SITES_PER_PAGE": "Sitios por página",
+      "NO_SITES_MESSAGE":"No eres miembro de ningún sitio. Por favor, póngase en contacto con su administrador para solicitar el acceso.",
```
Comment (rart) [inline static-assets/scripts/resources/locale-es.json]: This is mixing two types of language (formal/friendly) - tuteo with ustedeo.

@sumerjabri which tone do we want to take casual/friendly or more formal/respectful?

**Review by rart — CHANGES_REQUESTED** [review]
@sumerjabri see comment.

File: static-assets/scripts/resources/locale-es.json (hunk 1 above)
Comment (sumerjabri) [reply static-assets/scripts/resources/locale-es.json]: What's the tone of the rest of it?

If unclear, use formal.

Comment (vestrada) [reply static-assets/scripts/resources/locale-es.json]: Done

**Review by rart — APPROVED** [review]

**Pull request merged**